src/main/java/sx/blah/discord/handle/impl/obj/Channel.java

```diff
@@ -176,18 +180,18 @@ public void addToCache(IMessage message) {
 		}
 	}
 
-	private IMessage[] requestHistory(Long before, int limit) {
+	public IMessage[] requestHistory(Long before, Long after, int limit) {
```

Comment:
Practically, yes, but theoretically -1L could represent a valid uint64 message. I left it as is; Austin wrote it with the boxed long. It doesn't really cause any significant overhead etc, so...